lib/redis.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const error = require('./error')
+
+module.exports = (config, log) => {
+  const redis = require('fxa-shared/redis')(config, log)
+  if (! redis) {
+    return
+  }
+
+  return Object.entries(redis).reduce((object, [ key, value ]) => {
+    if (typeof value === 'function') {
+      object[key] = async (...args) => {
+        try {
+          return await value(...args)
+        } catch (err) {
+          if (err.message === 'redis.watch.conflict') {
+            // If you see this line in a stack trace in Sentry
+            // it's nothing to worry about, just a sign that our
+            // protection against concurrent updates is working
+            // correctly.
+            throw error.unexpectedError()
```

Comment:
It looks like the [shared redis code logs the conflict key already](https://github.com/mozilla/fxa-shared/blob/master/redis/connection.js#L72) so we should see that in the logs.